### PR TITLE
Add cron pattern option to create project command

### PR DESCRIPTION
**Pull request by aaronbauman**
Addresses https://github.com/pantheon-systems/terminus-build-tools-plugin/issues/394

### CODE REVIEW

File: src/Commands/ProjectCreateCommand.php [hunk 1]
```diff
@@ -463,7 +466,8 @@ function ($state) use ($ci_env, $siteDir, $use_ssh) {
             ->progressMessage('Beginning CI testing')
             ->taskCIStartTesting()
                 ->provider($this->ci_provider)
-                ->environment($ci_env);
+                ->environment($ci_env)
+                ->setCluCronSchedule($cluCronPattern);
```
Comment (greg-1-anderson) [inline src/Commands/ProjectCreateCommand.php:470]: The CLU cron schedule pattern is exactly the sort of configuration data that the $ci_env was intended to store. The parameter list for `startTesting` et. al. would get quite complicated if we had to list every configuration value as a parameter in the call stack. If you set a value on $ci_env here, then you should be able to fetch it by its key in startTesting, where it is needed.

Comment (aaronbauman) [reply src/Commands/ProjectCreateCommand.php:470]: ah, great, thanks for the feedback.
i'll look into putting the param into ci_env instead.

File: src/Commands/ProjectCreateCommand.php [hunk 2]
```diff
@@ -251,6 +250,8 @@ public function createProject(
         // Get the environment variables to be stored in the CI server.
         $ci_env = $this->getCIEnvironment($options['env']);
 
+        $ci_env->set('ci', 'clu-cron-pattern', $options['clu-cron-pattern']);
```
[the remark this answers is not in the document]
Comment (greg-1-anderson) [reply src/Commands/ProjectCreateCommand.php:253]: I don't quite remember the difference between `set` and `storeState`, but it looks like other code is using `storeState`. I can look later, or maybe @kporras07 could help out.

[the remark this answers is not in the document]
Comment (aaronbauman) [reply src/Commands/ProjectCreateCommand.php:253]: Yeah... this doesn't work ATM.
getting:
``` [error]  Tried to set state for a nonexistant owner 'ci'. ```

[the remark this answers is not in the document]
Comment (greg-1-anderson) [reply src/Commands/ProjectCreateCommand.php:253]: Try using storeState like the following lines; `set` is a lower-level construct iirc.

[the remark this answers is not in the document]
Comment (aaronbauman) [reply src/Commands/ProjectCreateCommand.php:254]: OK, Not sure that's the best way to do it, but `setState` immediately before the `set` call works.

**Pull request merged**